src/content/docs/en/guides/deploy/cloudflare.mdx

```diff
@@ -137,3 +137,14 @@ If you're encountering errors, double-check the version of `node` you're using l
 Cloudflare requires [Node v16.13](https://miniflare.dev/get-started/cli#installation), which is a more recent version than Astro’s out-of-the-box minimum, so double check you’re using at least v16.13.
 
 Client-side hydration may fail as a result of Cloudflare's Auto Minify setting. If you see `Hydration completed but contains mismatches` in the console, make sure to disable Auto Minify under Cloudflare settings.
+
+If you are building a project that is using server-side rendering and the server fails to build with error messages similar to:
+
+`[Error] Could not resolve "XXXX` `The package "XXXX" wasn't found on the file system but is built into node. Are you trying to
+  bundle for node? You can use "platform: 'node'" to do that, which will remove this error.`
+
+This means that a package or import you are using in the server-side environment is not compatible with the [Cloudflare Workers runtime APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/). 
+
+If you are directly importing a Node.js runtime API, please refer to the Astro documentation on Cloudflare's [Node.js compatibility](/en/guides/integrations-guide/cloudflare/#nodejs-compatibility) for further steps on how to resolve this.
+
+If you are importing a package that imports a Node.js runtime API, it is recommended that you check with the author of the package to see if they support the `node:*` import syntax. If they do not, you may need to find an alternative package or use a different adapter.
```

Comment:
```suggestion
If you are building a project that is using on-demand rendering with [the Cloudflare SSR adapter](/en/guides/integrations-guide/cloudflare/) and the server fails to build with an error message such as `[Error] Could not resolve "XXXX. The package "XXXX" wasn't found on the file system but is built into node.`:

- This means that a package or import you are using in the server-side environment is not compatible with the [Cloudflare Workers runtime APIs](https://developers.cloudflare.com/workers/runtime-apis/nodejs/). 

- If you are directly importing a Node.js runtime API, please refer to the Astro documentation on Cloudflare's [Node.js compatibility](/en/guides/integrations-guide/cloudflare/#nodejs-compatibility) for further steps on how to resolve this.

- If you are importing a package that imports a Node.js runtime API, check with the author of the package to see if they support the `node:*` import syntax. If they do not, you may need to find an alternative package or use a different adapter.
```
I shortened the original error message copying as it was a bit much and mostly the helpful info is upfront. Also, the "if" is only the start of a huuuuge sentence, so I think bullet points will help here. Not awesome maybe, but gets the point across!